server/sonar-web/src/main/js/apps/create/components/OrganizationAvatarUrlInput.tsx

```diff
@@ -40,6 +42,8 @@ interface State {
 export default class OrganizationAvatarUrlInput extends React.PureComponent<Props, State> {
   state: State = {error: undefined, editing: false, touched: false, value: ''};
   whiteListDomains: string[] = [];
+  amazon = "AMAZON";
```

Comment:
This constant is defined in multiple places.
Instead can you define in one common place.

Reply:
Please suggest the ideal place where the constant can be added. @jagadeshpaladugula

Reply:
Made the requested changes. Please review @jagadeshpaladugula